src/psyclone/tests/psyir/transformations/loop_trans_test.py

```diff
@@ -71,13 +71,6 @@ def test_loop_trans_validate(monkeypatch):
     # Check that validate is OK with a valid loop
     loop = invoke.schedule.walk(Loop)[0]
     trans.validate(loop)
```

Comment:
pylint says that `monkeypatch` is no longer used in this test. Since there's only one other complaint (f-string at L138) would you mind fixing that too?

Reply:
Done both